fv3core/tests/savepoint/translate/translate_fillz.py

```diff
@@ -48,12 +48,16 @@ def compute(self, inputs):
         for name, value in tuple(inputs.items()):
             if hasattr(value, "shape") and len(value.shape) > 1 and value.shape[1] == 1:
                 inputs[name] = self.make_storage_data(
-                    pad_field_in_j(value, self.grid.njd)
+                    pad_field_in_j(
+                        value, self.grid.njd, backend=self.grid.stencil_factory.backend
+                    )
```

Comment:
I don't think jeremy's PR altered the test code to not use global backend in the tests (and device_sync), so this is a step further!